x-pack/plugins/apm/public/hooks/use_apm_router.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useRouter } from '@kbn/typed-react-router-config';
+import type { ApmRouter } from '../components/routing/apm_route_config';
+import { useApmPluginContext } from '../context/apm_plugin/use_apm_plugin_context';
+
+export function useApmRouter() {
+  const router = useRouter();
+  const { core } = useApmPluginContext();
+
+  const link = (...args: any[]) => {
+    // @ts-ignore
```

Comment:
Why are we ignoring this? Can there be a comment afterwards?

Reply:
Yeah, TS will complain about a bunch of stuff that's hard to fix and doesn't really add a lot of value for a simple pass-through. Added a comment.